2-mandatory/4-people-i-know.js

```diff
@@ -403,8 +417,16 @@ This time, I only want the full names of the people who can multitask
 
 let colleaguesWhoCanMultitask = [];
 
+friends.forEach((friend) => {
+  friend.colleagues
+    .filter((colleague) => colleague.skills.includes("Multi-tasking"))
+    .forEach((colleague) =>
+      colleaguesWhoCanMultitask.push(`${colleague.name}`)
```

Comment:
amazing job with this one! 
You don't need to use string interpolation here as `colleague.name` is already a string